### PR TITLE
#165398196 : Feedback implementation (Day 2)

### PR DESCRIPTION
**Pull request by nkpremices**
#### What does this PR do?
Have the controllers, the readme and the validation schemas improved

#### Description of Task to be completed?
* Improve the validation of "FirstName" and "lastName" in the signup endpoint
* Insert information on the readme that an admin is created when the server starts
* Insert information on the readme that an admin is created when the server starts
* Add success messages on every success result 
* On login, show the type of user
* Update the auth testing file
* Update status values on the documentation

#### How should this be manually tested?
* Go to the documentation and test the endpoints with the required json formats to see the success messages
* Test the login endpoint to see the the type of user

#### What are the relevant pivotal tracker stories?
#165150240
#165150202
#165150160
#165150148
#165150128
#165150128
#165149953

### CODE REVIEW

File: server/helpers/v1/create.admin.js [hunk 1]
```diff
@@ -16,5 +16,5 @@ export default async () => {
     const tempUser = await usersModel.saveUser(admin.email,
         admin.firstName, admin.lastName, hashedPass,
         admin.type, admin.isAdmin, false);
-    console.log('admin created...', tempUser.id);
+    // console.log('admin created...', tempUser.id);
```
Comment (nakimera) [inline server/helpers/v1/create.admin.js]: remove commented out code plus console logs from your production code

File: server/helpers/v2/db.connector.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+pool.on('connect', () => {
+    console.log('connected to the db');
+});
+
+const querryDb = {
+    query(text, params) {
+        return new Promise((resolve, reject) => {
+            pool.query(text, params)
+                .then((res) => {
+                    // console.log(text);
```
Comment (nakimera) [inline server/helpers/v2/db.connector.js]: remove this too

File: server/middlewares/initialize.db.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import querryDb from '../helpers/v2/db.connector';
+import tables from '../storage/db';
+import queries from '../helpers/v2/db.querries';
+
+const admin = [
+    'premices@gmail.com',
```
Comment (nakimera) [inline server/middlewares/initialize.db.js]: consider using general names e.g
admin@gmail.com
firstName

File: server/middlewares/initialize.db.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    'premices',
+    'tuverer',
+    'dddddd4U',
```
Comment (nakimera) [inline server/middlewares/initialize.db.js]: consider using a general password eg `Password1`

File: README.md [hunk 5]
```diff
@@ -109,6 +111,10 @@ All the created API endpoints are documented and available [here](https://banka-
 * ```SALTING_ROUNDS```: Integer used to hash passwords
 * ```ADMIN_TOKEN, ADMIN_EMAIL, ADMIN_PASSWORD```: Admin token email and password for testing purposes
 
+### 3. Notes
+* When the server starts, a script creates the first admin user. He will be the one to create staff users and other admin users as well.
```
Comment (nakimera) [inline README.md:115]: what credentials can a person using your app use to log in as that admin

File: server/helpers/v2/db.connector.js (hunk 2 above)
Comment (nkpremices) [reply server/helpers/v2/db.connector.js]: Noted. I'm removing them all

File: server/middlewares/initialize.db.js (hunk 3 above)
Comment (nkpremices) [reply server/middlewares/initialize.db.js]: Well noted. Changing the current values to the new ones

File: server/middlewares/initialize.db.js (hunk 4 above)
Comment (nkpremices) [reply server/middlewares/initialize.db.js]: Noted

Comment (nkpremices) [reply server/middlewares/initialize.db.js]: Thank you

File: README.md (hunk 5 above)
Comment (nkpremices) [reply README.md:115]: Oh!! I'm going to add the new ones. According to the previous comments.

Comment (nkpremices) [reply README.md:115]: Thank you for the feedabacks

**Pull request merged**